fuzz/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
This is for fuzzing of the requests, meaning we can check invalid requests to see if they cause panics